scripts/python/engr_mode.py

```diff
@@ -15,264 +15,554 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
```

Comment:
The engr_mode module is being replaced with Ansible playbooks.  Please remove it from the pull request.